recipes/pycryptodome/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "pycryptodome" %}
+{% set version = "3.4.7" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "18d8dfe31bf0cb53d58694903e526be68f3cf48e6e3c6dfbbc1e7042b1693af7" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+
+  run:
+    - python
+
+test:
+  imports:
+    - Crypto
+    - Crypto.Cipher
+    - Crypto.Hash
+    - Crypto.IO
+    - Crypto.Math
+    - Crypto.Protocol
+    - Crypto.PublicKey
+    - Crypto.Random
+    - Crypto.SelfTest
+    - Crypto.SelfTest.Cipher
+    - Crypto.SelfTest.Hash
+    - Crypto.SelfTest.IO
+    - Crypto.SelfTest.Math
+    - Crypto.SelfTest.Protocol
+    - Crypto.SelfTest.PublicKey
+    - Crypto.SelfTest.Random
+    - Crypto.SelfTest.Signature
+    - Crypto.SelfTest.Util
+    - Crypto.Signature
+    - Crypto.Util
+
+about:
+  home: http://www.pycryptodome.org
+  license_file: {{ RECIPE_DIR }}/LICENSE.rst
```

Comment:
Please submit a PR upstream to add `LICENSE.rst` to the `MANIFEST.in` for future distribution. Also please link that PR in a comment in the recipe above the `license_file` line. Thanks.

Reply:
Double-checked the source tarball, and it turns out a license is already included. No need for a manifest.